src/api/java/Sort.java

```diff
@@ -27,25 +27,20 @@
 {
     /**
      * Equality operator for objects of type Sort. 
-     * @param o
-     * @return
```

Comment:
It's not about not removing information, but about adding useful information. It's fine though, just remind me when you're done and I'll do it myself.